src/App.jsx

```diff
@@ -1,9 +1,89 @@
-import './App.css'
+import './App.css';
+import Button from './components/button/Button.jsx';
+import bag_1 from './assets/bag_1.png';
+import bag_2 from './assets/bag_2.png';
+import bag_3 from './assets/bag_3.png';
+import bag_4 from './assets/bag_4.png';
+import brand from './assets/brand.png';
+import our_story from './assets/our_story.png';
+import Product from './components/product/Product.jsx';
+import Tile from './components/tile/Tile.jsx';
+
+const buttonTexts = [
```

Comment:
Het gebruik van een Array om daarin de namen van de buttons te zetten is wat omslachtig. Wanneer het wel nuttig zou zijn geweest is als je de map methode had toegepast op de array.